docs/TypeScript.md

```diff
@@ -2,10 +2,17 @@
 
 <a id="typescript"></a>
 ## TypeScript
-Fastify is shipped with the a typings file so it should "just work" when used in a TypeScript application.
+Fastify is shipped with the a typings file so it should "just work" when used in a TypeScript application.<br/>
+*You should also install `@types/node` and `@types/pino`.*
```

Comment:
can you make it explicit

npm i @types/node @types/pino

Reply:
`@types/node` isn't always necessary. Typescript projects only need it for accessing node built-in classes. Most projects do need the built-in classes, so Typescript programmers are used to installing `@types/node` first thing as a matter of habit.

Reply:
It's not yet clear to me whether either `@types/node` or `@types/pino` is actually needed in `devDependencies` for Fastify. I'll look into that this weekend.

Reply:
Since Fastify ships with a particular version of pino, it would make sense for `@types/pino` to stay in `dependencies` so that the pino version and its typings will stay in sync in this repo.

Reply:
We try to ship always the latest.
I don't want to force not-typescript users to download useless stuff :)

Reply:
That makes sense @nwoltman. Spares TS devs from figuring out the right version of typings to use.

Reply:
@delvedor I don't want to force not-TypeScript users to download useless stuff either, but we're already doing that by including `fastify.d.ts`, which has a direct dependency on `@types/pino`.

TypeScript users should include `@types/node` themselves because they choose which version of Node to using, but since `fastify` depends on `pino`, it is `fastify` that chooses which version of `pino` to use.

We won't have to worry about this though if pino includes its own typings, which might happen soon (see [this discussion](https://github.com/pinojs/pino/issues/344#issuecomment-358049674) in the pino repo).

Reply:
I'd be careful with that word "soon". Unless someone steps up with at least a PR, there will not be type in the `pino` module any time soon.